Sources/MuxStatsGoogleIMAPlugin/include/module.modulemap

```diff
@@ -1,4 +1,8 @@
 
 module MuxStatsGoogleIMAPlugin {
     header "MuxImaListener.h"
+    
+    explicit module Private {
+        header "MuxImaListener+Private.h"
```

Comment:
Out of curiosity, what happens if this isn't present?

Reply:
If this isn't present, the SPM test build fails because there's no module called `MuxStatsGoogleIMAPlugin.Private`.

If you added `MuxImaListener+Private.h` as a `header` declaration, then it would go in the same module as everything else, and be visible with the same `import`